src/core/dune.cpp

```diff
@@ -30,15 +30,18 @@
 #include <dune/common/parallel/mpihelper.hh>
 #include <dune/common/parametertree.hh>
 #include <dune/common/parametertreeparser.hh>
-#include <dune/copasi/enum.hh>
-#include <dune/copasi/gmsh_reader.hh>
-#include <dune/copasi/model_diffusion_reaction.cc>
-#include <dune/copasi/model_diffusion_reaction.hh>
-#include <dune/copasi/model_multidomain_diffusion_reaction.cc>
-#include <dune/copasi/model_multidomain_diffusion_reaction.hh>
-#include <dune/grid/io/file/gmshreader.hh>
+#include <dune/copasi/common/enum.hh>
+#include <dune/copasi/grid/mark_stripes.hh>
+#include <dune/copasi/grid/multidomain_gmsh_reader.hh>
+#include <dune/copasi/model/base.hh>
+#include <dune/copasi/model/diffusion_reaction.cc>
+#include <dune/copasi/model/diffusion_reaction.hh>
+#include <dune/copasi/model/multidomain_diffusion_reaction.cc>
```

Comment:
If you include the library, these `.cc` are not necessary.
 ```suggestion
```

Reply:
thanks! I'll remove them